.github/workflows/ansible-test.yml

```diff
@@ -1,25 +1,67 @@
 name: CI
```

Comment:
@rrey I think the change to `.github/workflows/ansible-test.yml` can be reverted. 

I just did a diff between your file and https://github.com/gundalow-collections/grafana/pull/15/files and I don't see any other changes I need to pull in.